src/viam/sessions_client.py

```diff
@@ -44,24 +41,24 @@ class SessionsClient:
 
     _current_id: str = ""
     _disabled: bool = False
-    _supported: Optional[bool] = None
     _heartbeat_interval: Optional[timedelta] = None
 
-    def __init__(self, channel: Channel, *, disabled: bool = False):
+    def __init__(self, channel: Channel, address: str, dial_options: Optional[DialOptions], *, disabled: bool = False):
         self.channel = channel
         self.client = RobotServiceStub(channel)
         self._disabled = disabled
-        self._lock = asyncio.Lock(**loop_kwargs())
+
+        self._address = address
+        self._dial_options = dial_options
+
+        # can only synchronize numbers and chars across processes, so 0 = Unknown, 1 = True, 2 = False
```

Comment:
let's make these values consts or enums to codify this

Reply:
done